README.md

```diff
@@ -1,10 +1,12 @@
-# YoutubeArchiver
+# Overview
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/youtubearchiver`. To experiment with that code, run `bin/console` for an interactive prompt.
+YoutubeArchiver is a Ruby gem that downloads YouTube Video metadata and media. It works in conjunction with [Zenodotus](https://github.com/TechAndCheck/zenodotus) and [Hypatia](https://github.com/TechAndCheck/hypatia) to archive fact-checked image/video posts. YoutubeArchiver exists alongside a collection of other media scrapers created by the Duke Reporter's Lab, including [Birdsong](https://github.com/cguess/birdsong/) (a Twitter scraper), [Zorki](https://github.com/cguess/zorki) (an Instagram scraper), and [Forki](https://github.com/TechAndCheck/forki) (a Facebook scraper). 
```

Comment:
A quick typo, but we work in a journalism lab so can't help myself. It's "Duke Reporters' Lab" (note the apostrophe)